packages/react-components/src/organisms/TeamMembersTabbedCard.tsx

```diff
@@ -40,7 +40,10 @@ const TeamMembersTabbedCard: React.FC<TeamMembersTabbedCardProps> = ({
 }) => {
   const [alumniMembers, activeMembers] = splitListBy(
     members,
-    (member) => isTeamInactive || !!member?.alumniSinceDate,
+    (member: TeamMember) =>
+      isTeamInactive ||
+      !!member?.alumniSinceDate ||
+      member?.status === 'Inactive',
```

Comment:
Since this `status` is a required field we don't need `?` here 😉 
```suggestion
      member.status === 'Inactive',
```